charts/quantum-serverless/charts/gateway/templates/rayclustertemplate.yaml

```diff
@@ -141,21 +141,84 @@ data:
                 - name: OT_ENABLED
                   value: {{ .Values.application.ray.openTelemetryCollector.enabled | quote }}
 {{- end }}
-            - image: fluent/fluent-bit:1.9.10
-              name: ray-head-logs
+            - image: proxy:local
+              #imagePullPolicy: IfNotPresent
+              #name: tcpdump-sidecar
+              #command: ["/bin/bash"]
+              #args: ["-c", "while true; do ping localhost; sleep 60;done"]
+              securityContext:
+                runAsGroup: 123
+                privileged: true
```

Comment:
Is it possible to not run the proxy as privileged?

Reply:
Yes, I think so.  The setting of the iptables must be separated out into an init container then the proxy can run in none-privileged.  That is one of TODOs.